content/docs/gke/deploy/deploy-ui.md

```diff
@@ -51,4 +47,60 @@ takes longer than expected, try accessing the above URI anyway.
   [upgrade or reinstall a Kubeflow Pipelines 
   deployment](/docs/pipelines/upgrade/).
 * [Troubleshoot](/docs/gke/troubleshooting-gke) any issues you may
-  find.
\ No newline at end of file
+  find.
+  
+  <!--Sub-sections, these will be not be part of this page  -->
+## Login with GCP Iap {:#Login_with_GCP_IAPSection}
+If you select this option, you see the form:
+<img src="/docs/images/Login with GCP IAP option.png" 
+  alt="Login with GCP IAP option"
+  class="mt-3 mb-3 border border-info rounded">
+- If you haven’t already done so, create an [OAuth Client](https://www.kubeflow.org/docs/gke/deploy/oauth-setup/) and enter the client ID and secret in the field.
+- The default GKE zone is us-central-1a. Use the dropdown to select the zone you want.
+- The Kubeflow version cannot be changed currently.
+
+Note: The three buttons at the bottom of the form are:
+- **Create Deployment**: Click this to deploy your Kubeflow project.
```

Comment:
This list does not render properly. See preview:
https://deploy-preview-685--competent-brattain-de2d6d.netlify.com/docs/gke/deploy/deploy-ui/

Try adding an empty line before the first list item.

Reply:
added line, looks good now